.circleci/config.yml

```diff
@@ -9,14 +9,17 @@ jobs:
     executor:
       name: hmpps/java
       tag: "11.0"
+    environment:
+      _JAVA_OPTIONS: -Xmx512m -XX:ParallelGCThreads=2 -XX:ConcGCThreads=2 -Djava.util.concurrent.ForkJoinPool.common.parallelism=2 -Dorg.gradle.daemon=false
```

Comment:
This improves the memory management when running the tests.  I was starting to see out of memory errors.